js/BUILDING.md

```diff
@@ -1,184 +1,121 @@
-
-On macOS the first time you run the tests, you will be prompted for your
-password. This is necessary to configure the trust setting for the HTTP
-server certificate, which will enable you to connect to the HTTP server
-with SSL via your web browser.
-
-### Browser Information
+## Creating the NPM package
 
-#### Self-Signed Certificate
-
-The browser-based tests allow you to choose whether to run the tests over
-non-secure WebSocket (WS) or secure WebSocket (WSS) connections. The WSS
-connections in these tests rely on a self-signed certificate, `ca_cert.pem`,
-provided in the certs(../certs) directory.
-
-To successfully run the tests over WSS, additional action may be necessary
-depending on the browser you're using:
-
-- Chrome
-   You'll be presented with a warning about the site's security certificate.
-   Click the "Proceed anyway" button to temporarily accept the certificate.
-
-- Firefox
-   You'll see a warning saying "This Connection is Untrusted". Open Firefox's
-   Preferences or Options dialog, click on the Advanced section, select the
-   Certificates tab and click on the "View Certificates..." button. In the
-   Authorities tab, click the "Import..." button, navigate to the `ca_cert.pem`
-   file, and add it as a certificate authority (CA) for trusting web sites.
-   After closing the dialogs, reload the test page to continue. You should
-   uninstall this certificate after running the tests.
-
-- Microsoft Edge
-   Run the management console (mmc.exe) and add the Certificates snap-in for
-   the computer account. Then select Console Root > Certificates (Local
-   Computer) > Trusted Root Certificate Authorities. In the Action menu, choose
-   All Tasks and Import. Navigate to the `ca_cert.pem` file and import it into
-   the Trusted Root Certificate Authorities. Reload the test page to continue.
-   You should uninstall this certificate after running the tests.
-
-#### Secure WebSockets on iOS and Android
-
-To use WSS on iOS and Android it may be required (depending on browser and
-platform) that the server certificate's common name matches the computer hosting
-the tests, and that the test certificate authority be installed on your device.
-
-First you'll need to generate new certificates to match the IP address of the
-computer hosting the tests:
+To create the `@zeroc/ice` NPM package, open a command prompt and run the following command:
 
 ```shell
-certs/makecerts.py [IP address]
+npm pack
 ```
 
-Next you must install the certificate authority on your device. The simplest way
-is to email the CA certificate (`certs/cacert.pem`) to yourself and then follow
-the instructions below to install the certificate on your device.
-
-Once installed, you connect to the server using the same IP address used to
-create the certificates.
+### Slice compilers
 
-##### Installing Certificates on iOS
+By default, the NPM package `@zeroc\ice` includes only the `slice2js` compiler created by the local C++ build.
```

Comment:
```suggestion
By default, the NPM package `@zeroc/ice` includes only the `slice2js` compiler created by the local C++ build.
```

Reply:
Seems there is a problem with GitHub, and it is not displaying the latest commits in the PR.

Reply:
Weird!